TechItEasy.sql

```diff
@@ -0,0 +1,95 @@
+DROP TABLE IF EXISTS Users;
+DROP TABLE IF EXISTS Television_WallBracket;
+DROP TABLE IF EXISTS WallBrackets;
+DROP TABLE IF EXISTS RemoteControllers;
+DROP TABLE IF EXISTS Televisions;
+DROP TABLE IF EXISTS CIModules;
+DROP TABLE IF EXISTS attachtmentMethods;
+DROP TABLE IF EXISTS Products;
+
+CREATE TABLE Users (
```

Comment:
Persoonlijk zou ik het aantal toegestane tekens in de string groter maken dan 30